File: module_gan/src/main/java/com/huyingbao/module/gan/ui/main/action/MainActionCreator.java
package com.huyingbao.module.gan.ui.main.action;

import com.huyingbao.core.arch.action.RxActionCreator;
import com.huyingbao.core.arch.action.RxActionManager;
import com.huyingbao.core.arch.dispatcher.RxDispatcher;
import com.huyingbao.core.arch.scope.ActivityScope;

import javax.inject.Inject;


/**
 * rxAction创建发送管理类
 * Created by liujunfeng on 2019/1/1.
 */
@ActivityScope
public class MainActionCreator extends RxActionCreator implements MainAction {
    @Inject
    MainActionCreator(RxDispatcher rxDispatcher, RxActionManager rxActionManager) {
        super(rxDispatcher, rxActionManager);
    }
}

File: module_gan_kotlin/src/main/java/com/huyingbao/module/gan/ui/main/action/MainActionCreator.kt
package com.huyingbao.module.gan.ui.main.action

import com.huyingbao.core.arch.action.RxActionCreator
import com.huyingbao.core.arch.action.RxActionManager
import com.huyingbao.core.arch.dispatcher.RxDispatcher
import com.huyingbao.core.arch.scope.ActivityScope

import javax.inject.Inject


/**
 * rxAction创建发送管理类
 * Created by liujunfeng on 2019/1/1.
 */
@ActivityScope
class MainActionCreator @Inject
internal constructor(rxDispatcher: RxDispatcher, rxActionManager: RxActionManager) : RxActionCreator(rxDispatcher, rxActionManager), MainAction
